source/StepBro.Core/Execution/ScriptUtils.cs

```diff
@@ -290,6 +347,13 @@ public static string Await(this ILineReader reader, [Implicit] ICallContext cont
                     }
                 }
 
+                // If the current entry in the reader has a timestamp after our limit
+                // we break as we know that any entry after this would have been too late
+                if (reader.Current != null && reader.LinesHaveTimestamp && reader.Current.Timestamp > to)
```

Comment:
I think this should be removed, as the Await does not have a timestamp limit time.